src/Apps/Artist/Routes/Overview/state.ts

```diff
@@ -40,6 +41,48 @@ export const initialState = {
   showZeroState: false,
 }
 
+// Returns a string representing the query part of a URL.
+// It removes default values.
+export const urlFragmentFromState = (state: State, extra?: Partial<State>) => {
```

Comment:
This function, and `isDefaultFilter` just below it, are really similar to functions in the Collect/Search Apps. I think the duplication is correct - while the functions are similar, they aren't identical, and each App seems like it might need to do each of these things slightly differently.